wasm-mappings/README.md

```diff
@@ -0,0 +1,33 @@
+# `source-map-mappings`
```

Comment:
Sould this become `wasm-mappings`?

Reply:
No, the rust package name is still source-map-mappings (you can see it referenced in cargo files)